typescript/infra/config/environments/dev/index.ts

```diff
@@ -1,18 +1,16 @@
-import { getMultiProviderFromGCP } from '../../../scripts/utils';
-import { CoreEnvironmentConfig } from '../../../src/config';
-import { Contexts } from '../../contexts';
+// import { getMultiProviderFromGCP } from '../../../scripts/utils';
```

Comment:
why not remove this and the rest of the `dev` directory?

Reply:
I was originally thinking to keep it around if we will imminently create the dev environment but maybe thats too optimisitic, so just removed for now